backend/Auth/ResetPassword.ts

```diff
@@ -0,0 +1,23 @@
+import { router } from "../Router.js";
+import { body } from "express-validator";
+import { sendResetPasswordEmail } from "../Utils/EmailUtils.js";
+import { findUserByEmail } from "../Users/Users";
+
+// send email to user with link to reset password
+router.post(
+  "/send-reset-password-email",
+  body("email").isEmail(),
```

Comment:
When using express validator, you have to do an if statement to check if there were any validation errors. Please add that to this endpoint

Reply:
Ah, I think I understand now what you meant by needing to send the 400 error manually.  Thank you.